finals-2024-01-16/code/5332-milana-sokolova/src/app/app.component.html

```diff
@@ -1,4 +1,17 @@
-<app-header></app-header>
-<router-outlet></router-outlet>
-<app-footer [student]="student$ | async" ></app-footer>
-
+<header>
```

Comment:
While this is a header and a footer, it was explicitly specified that we need them as components. +2 points for the functionallity